barretenberg/cpp/src/barretenberg/flavor/flavor.hpp

```diff
@@ -441,6 +447,25 @@ template <typename T> concept IsFoldingFlavor = IsAnyOf<T, UltraFlavor,
                                                             MegaRecursiveFlavor_<CircuitSimulatorBN254>,
                                                             MegaZKRecursiveFlavor_<MegaCircuitBuilder>,
                                                             MegaZKRecursiveFlavor_<UltraCircuitBuilder>>;
+#else
+template <typename T> concept IsFoldingFlavor = IsAnyOf<T, UltraFlavor,
```

Comment:
Looks like maybe this concept is not new to this PR? `IsFoldingFlavor` is not a useful or correct name here. Not sure what this concept is supposed to encompass